examples/README.md

```diff
@@ -23,3 +23,7 @@ See [spring-boot-kubernetes](spring-boot-kubernetes) for containerizing a [Sprin
 ### Micronaut example
 
 See [micronaut](micronaut) for containerizing a [Micronaut framework](https://micronaut.io/) Groovy/Java application.
+
+### Java Agents example
+
+See [java-agent](java-agent) for launching with a Java agent.
```

Comment:
Might be good to include the specific one used:

`...launching with a Java agent (Stackdriver Debugger).`